src/java/org/apache/cassandra/service/accord/async/AsyncOperation.java

```diff
@@ -124,8 +124,9 @@ public AsyncOperation(AccordCommandStore commandStore, PreLoadContext preLoadCon
         this.commandStore = commandStore;
         this.preLoadContext = preLoadContext;
         this.loader = createAsyncLoader(commandStore, preLoadContext);
-        setLoggingIds();
```

Comment:
mind reverting this class?  your logic is no longer here

Reply:
Reverted, thanks.